packages/fxa-settings/.storybook/main.js

```diff
@@ -4,7 +4,7 @@
 
 module.exports = {
   stories: ['./design-guide/main.stories.tsx', '../src/**/*.stories.tsx'],
-  staticDirs: ['./design-guide'],
+  staticDirs: ['../public'],
```

Comment:
I couldn't spot any issue created by removing this folder, but just double checking that we don't need it. There are some assets in fxa-settings/.storybook/design-guide that don't exist in public.

Reply:
This is a good call, it might be needed for the deployed version

Reply:
The deployed storybook works without including `./design-guide` in the static Dirs - however the localization does not, even when including the public folder. I don't think that's a requirement to complete this ticket, but I'll file a follow-up ticket.

Reply:
Got the l10n to work on the deployed SB 💃  Turns out I had just missed a period in the path...